source/install-desktops/customize-desktop-app.rst

```diff
@@ -0,0 +1,34 @@
+.. _install-desktops-customize-desktop-app:
+
+Customize Desktop App
+=====================
+
+In some cases the defaults used to submit a batch job may not work on your
+given cluster or you may just want to remove some form options presented to the
+user and fill in hard-coded values. All of these cases and more can easily be
+customized in the following files:
+
+:file:`bc_desktop/local/{cluster_desktop}.yml`
```

Comment:
The `{...}` signifies it is a variable. Whereas previously I was using `cluster1.yml` as the actual file name.

Also I didn't want to confuse people thinking that the name of the yaml file under `bc_desktop/local` had to match the name of the file in cluster configs. As those names do not need to correspond to each other and I wanted to clear up any confusion that they might be the same file.